contrib/checkbox-ce-oem/checkbox-provider-ce-oem/data/video-codec-test-confs/genio-350.json

```diff
@@ -0,0 +1,14 @@
+{
+    "gst_v4l2_audio_video_synchronization": {
+        "video_sinks": {
+            "non_desktop": "kmssink connector-id=32 driver-name=mediatek"
+        },
+        "cases": [
+            {
+                "decoder_plugin": "v4l2h264dec",
+                "golden_sample_files": ["480p.mp4", "720p.mp4", "1080p.mp4"]
```

Comment:
Based on the spec [Rity 24](https://download.mediatek.com/aiot/download/release-note/v24.0/v24.0_IoT_Yocto_Feature_Table_v1.0.pdf) (page 22), the maximum resolution supported by G350 is 1920x1080, so we don't need the 4K case.